.env.example

```diff
@@ -4,9 +4,9 @@ COOKIE_NAME=sb-token
 SUPABASE_JWT_SECRET=
 COOKIE_SECRET=
 HCAPTCHA_SECRET=
-DIRECT_URL="postgresql://postgres:[YOUR_DB_PASSWORD]@db.[YOUR_SUPABASE_PROJECT_ID].supabase.co:5432/postgres"
-DATABASE_URL="postgresql://postgres:[YOUR_DB_PASSWORD]@db.[YOUR_SUPABASE_PROJECT_ID].supabase.co:6543/postgres?pgbouncer=true"
+DIRECT_URL="postgres://postgres.[YOUR_SUPABASE_PROJECT_ID]:[YOUR_DB_PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:6543/postgres"
+DATABASE_URL="postgres://postgres.[YOUR_SUPABASE_PROJECT_ID]:[YOUR_DB_PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:6543/postgres?pgbouncer=true"
```

Comment:
Might be an inconsequential nitpick, but the query parameters seem to be incorrect based on the string I'm seeing in the SB UI:
```
postgres://postgres:[YOUR-PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:6543/postgres?options=reference%3Dawyuaxlpezyhfkpznrox
```

Also we should probably remove the `pgbouncer=true` param even if it's a noop now to avoid confusion.

Reply:
Aware this is only the sample file but not sure if it got applied like this in the actual secrets

Reply:
Here's the [issue](https://github.com/prisma/prisma/issues/21514) for the pgbouncer hack and regarding the connection string it's the one from the connection pooling section. See screenshot below.
<img width="1204" alt="Capture d’écran 2024-01-19 à 16 03 59" src="https://github.com/WalletConnect/cloud-auth-api/assets/32299095/74c8463b-27d6-433a-a6ac-139eb78f0ee7">

Reply:
Actually the format for the connection string is different on the dev and prod DBs..